packages/forms-web-app/src/lib/is-token-valid.js

```diff
@@ -57,7 +57,11 @@ const isTokenValid = async (id, token, emailAddress, session) => {
 		action: ''
 	};
 
-	if (!id || typeof id !== 'string' || !token || typeof token !== 'string') return result;
+	if (!token || typeof token !== 'string') return result;
+
+	if ((!id || typeof id !== 'string') && (!emailAddress || typeof emailAddress !== 'string')) {
```

Comment:
Now's there are three of the same checks, worth a util for readability?

`emptyOrNotString` ?

Reply:
have gone with an isString util function